src/MinFraud/ReportTransaction.php

```diff
@@ -116,18 +121,27 @@ public function report(
             $this->verifyEmpty($values);
         }
 
+        if ($ipAddress === null
+            && $minfraudId === null
```

Comment:
Given that we are not validating that this is a valid UUID, maybe w should at least check that it is not the mepty string.

Reply:
`$minfraudId` is validated as a UUID a little later in the validation process
https://github.com/maxmind/minfraud-api-php/blob/a0f49df251a8d82edf8399bb3311300344f4ca1e/src/MinFraud/ReportTransaction.php#L153-L162

That being said, `$maxmindId` is also validated to ensure it is 8 characters so I don't think we need the check for `$maxmindId === ''`. It might also make sense to put this "one of these fields" check after the aforementioned individual field validation has occurred.